ts/features/itwallet/identification/common/screens/ItwIdentificationModeSelectionScreen.tsx

```diff
@@ -240,13 +254,20 @@ const SpidMethodModule = () => {
   }, [level]);
 
   return (
-    <ModuleNavigationAlt
-      title={title}
-      subtitle={subtitle}
-      testID="SpidMethodModuleTestID"
-      icon="spid"
-      onPress={handleOnPress}
-    />
+    <VStack space={16}>
+      {isReissuanceMode && (
+        <ItwIdentificationFrequencyRow
+          label={I18n.t(`${i18nNs}.frequency.every90Days`)}
+        />
+      )}
```

Comment:
After taking another look, I realized I misunderstood the component, so my previous suggestion was wrong. This logic does not belong to this component.
Please, can you revert this change? 
Thank you!